experimental/PlaneCurve/ParaPlaneCurves.jl

```diff
@@ -118,6 +175,42 @@ end
 Return a vector `V` defining the anticanonical map `C --> PP^(n-2)`. Note that the
 entries of `V` should be considered as representatives of elements in R/I,
 where R is the basering.
+
+# Example
+```jldoctest
+julia> R, (v, w, x, y, z) =  GradedPolynomialRing(QQ, ["v", "w", "x", "y", "z"])
+(Multivariate Polynomial Ring in v, w, x, y, z over Rational Field graded by 
+  v -> [1]
+  w -> [1]
+  x -> [1]
+  y -> [1]
+  z -> [1], MPolyElem_dec{fmpq, fmpq_mpoly}[v, w, x, y, z])
+
+julia> M = matrix(R, 2, 4, [v w x y; w x y z])
+[v   w   x   y]
+[w   x   y   z]
+
+julia> V = minors(M, 2)
+6-element Vector{MPolyElem_dec{fmpq, fmpq_mpoly}}:
+ v*x - w^2
+ v*y - w*x
+ w*y - x^2
+ v*z - w*y
+ w*z - x*y
+ x*z - y^2
+
+julia> I = ideal(R, V);
+
+julia> RNC = ProjCurve(I)
+Projective curve defined by the ideal(v*x - w^2, v*y - w*x, w*y - x^2, v*z - w*y, w*z - x*y, x*z - y^2)
+
```

Comment:
The extra space here occurs because `println` is used here. Both should just be `print`.
https://github.com/oscar-system/Oscar.jl/blob/45e437433522dca49be0cd48e056ff4395ae7453/experimental/PlaneCurve/ProjCurve.jl#L43-L48

Reply:
Indeed, we should address that in a later PR. There are many, many similar cases with extra trailing newlines.

Reply:
@tthsqe12 thx, changed in 3da9619